Workbooks/LogAnalytics Workspace/AtResource/Workspace Insights.workbook

```diff
@@ -0,0 +1,1336 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "crossComponentResources": [
+          "/subscriptions/5733bcb3-7fde-4caf-8629-41dc15e3b352/resourceGroups/CH-OpsRG-Pri/providers/Microsoft.OperationalInsights/workspaces/CH-LA"
```

Comment:
The build is still complaining that this is failing, can you remove `crossComponentResources`